OG_Format.adoc

```diff
@@ -113,7 +113,7 @@ This document defines the common OceanGliders data format, hereafter OG1.0.
 ////
 == Global attributes
 
-The global attribute section is used for data discovery. The following global attributes should appear in the global section. The NetCDF Climate and Forecast (CF) Metadata Conventions are available from: http://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#trajectory-data[_http://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#trajectory-data_]. It is recommended that extra global attributes follow ACDD convention".
+The global attribute section is used for data discovery. The following global attributes should appear in the global section. The NetCDF Climate and Forecast (CF) Metadata Conventions are available from: https://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#_single_trajectory[_https://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#_single_trajectory_]. It is recommended that extra global attributes follow ACDD convention".
```

Comment:
Do we need to move back to CF-1.8?

Reply:
@castelao - The latest CF compliance checker I see is for CF-1.8. If this is the case, I recommend using CF-1.8 so that data providers can easily verify their files are to spec.

Reply:
Is there any CF compliance checker for CF-1.10 available ?
Do we need a CF compliance checker or an OG1.0 checker ? 
I am asking as if we make sure the OG1.0 format is compliant with CF-1.10 we do not need to run CF compliance checker but only OG1.0 compliance checker. CF complaince would then be "included" in the OG1.0 compliance checker.
What do you think @jenseva ?

Reply:
As far as I know, the changes from CF-1.8 to CF-1.10 are irrelevant for OG. Thus, we can move to `1.10` and still use the 1.8 checker.

The example H.4.2 for CF-1.10 is just an example of one specific case. It does not require the trajectory to be a `char` type. It is valid and consistent to a 'single trajectory' to choose `trajectory` to be type `string`.

I suggest keeping the original reference to 'CF-1.10'.